### DISCUSSION TITLE
Do i need to stop while changing destination disks

### DISCUSSION THREAD

**Discussion by LazarusDarkside** [swar/Swar-Chia-Plot-Manager#944] [Q&A]
Hello,
I want to know do i need to stop plotting and giving a new job before changing destination disk? Or if i plug the full disk and put a new one with same drive name will be enough to continue?

**Comment by mbabak74** [comment]
Did you read the FAQ? Developer says it's not recommended.
But I think you can try. You only risk is to loose your current jobs, which you also loose when you stop plotting (I think waiting for al the plots to finish takes to much time. Just kill all the jobs that still need a couple of hours to finish and wait for the last ones to finish)

**Comment by Chilli71** [comment]
You need to understand that the manager kicks off the plots as individual processes.
When it kicks off the plot for this plot the parameters are fixed.
If you chnage the parameters new plots will kicked off with the new parameters.
If you have only one directory specified (for each temp/trmp2/final) chances that it works with resart are good. If you us multiple directories not sure what happens to the logic if you cuange it at runtime.
When you stop or restaet the manager the kicked off plots keep running. Changes of the parameters have no effect on the running plots.
